### PR TITLE
Create ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59716](https://cumulocity.atlassian.net/browse/MTM-59716)
Original PR: [6673](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6673)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: load title.decorator on loading application (#6621) [GRAFT][release/cd] (#6673)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix loading of title decorator when loading application
```

File: content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59716
+version: 1020.3.2
+---
+load title.decorator on loading application (#6621) [GRAFT][release/cd] (#6673)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md:17]: The Claude AI suggested to use the following body:
```suggestion
In certain scenarios, the title decorator was not properly loaded when an application was loaded, leading to missing or incorrect application titles being displayed to users. This issue has been resolved by ensuring the title decorator is consistently loaded whenever an application is loaded. Users will now see the correct application title displayed at all times.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by mmed-softwareag — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59716
+version: 1020.3.2
+---
+In certain scenarios, the title decorator was not properly loaded when an application was loaded, leading to missing or incorrect application titles being displayed to users. This issue has been resolved by ensuring the title decorator is consistently loaded whenever an application is loaded. Users will now see the correct application title displayed at all times.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md:17]:
```suggestion
In certain scenarios, the title decorator was not properly loaded when an application was loaded, leading to missing or incorrect application titles. This issue has been resolved by ensuring the title decorator is consistently loaded whenever an application is loaded. Users will now see the correct application title displayed at all times.
```

File: content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix loading of title decorator when loading application
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md:3]:
```suggestion
title: Fix missing or incorrect titles when loading application
```

File: content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed missing or incorrect titles when loading application
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-3-2-load-title-decorator-on-loading-application-6621-graft-release-cd-6673.md:3]:
```suggestion
title: Fixed missing or incorrect titles when loading applications
```

**Pull request merged**